tests/test_types.py

```diff
@@ -188,6 +188,13 @@ def test_boolean():
         Boolean(value=1, caption="caption")
 
 
+def test_filepath():
+    from alfasim_sdk.types import FilePath
+
+    file_path = FilePath(caption="Test")
+    assert file_path
```

Comment:
For a smoke test only creating the object is enough; `assert file_path` gives the impression that it might implement `__bool__`, which is not the case. Thus I suggest to use simply:

```python
def test_filepath():
    from alfasim_sdk.types import FilePath

    FilePath(caption="Test")    
```

Reply:
Please remove this `assert` 👍

Reply:
Solved ;)